actions.go

```diff
@@ -109,6 +109,9 @@ func ReadActionsYaml(r io.Reader) (*Actions, error) {
 		if valid := actionNameRule.MatchString(name); !valid {
 			return nil, fmt.Errorf("bad action name %s", name)
 		}
+		if reservedName(name) {
+			return nil, fmt.Errorf("reserved action name %s", name)
```

Comment:
I've formatted/worded it a little differently, but kept with the spirit of making the error message human-readable.